Banyan/src/queues.jl

```diff
@@ -40,7 +40,7 @@ end
 ###################
 
 function sqs_receive_message_with_long_polling(queue)
-    r = AWSSQS.sqs(queue, "ReceiveMessage", MaxNumberOfMessages = "1", WaitTimeSeconds = "20")
+    r = AWSSQS.sqs(queue, "ReceiveMessage", MaxNumberOfMessages = "1", ReceiveMessageWaitTime = "20")
```

Comment:
Has this been properly  tested, and do we know this is what we want, for reduced latency?

Reply:
This has been tested. I moved the wait time back to queue creation and I did confirm that long polling has desirable performance. It's usually not more than a second off in terms of returning early if a message is sent before the 20s is up.